### DISCUSSION TITLE
Windows Explorer "Bar"

### DISCUSSION THREAD

**Discussion by Regexxer405** [Open-Shell/Open-Shell-Menu#673] [Q&A]
I just installed Open Shell and I like it very much, thank you.

I think Open Shell added two bars to the Windows Explorer window (one at the top/middle and one at the bottom). How can I disable it?
It looks like this:
https://imgur.com/4yFbrah

The grey row at the bottom.

**Comment by bonzibudd** [comment]
If you run the installer again, you can uncheck the "Classic Explorer" option and that should remove it.

**Reply by Regexxer405** [reply]
Thank you, that worked! :)
